CMakeCommon.cmake

```diff
@@ -2,7 +2,7 @@ function (SetGlobalCompilerDefinitions acVersion)
 
     if (WIN32)
         add_definitions (-DUNICODE -D_UNICODE -D_ITERATOR_DEBUG_LEVEL=0)
-        set (CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL PARENT_SCOPE)
+        set (CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL CACHE STRING "" FORCE)
```

Comment:
I really don't like how project code sets `CMAKE_*` variables, but at the same time this is practically optional and we don't release the devkit with `/MDd` linkage, so the value set here is appropriate.